src/app/transactions/MempoolFeePieChart.tsx

```diff
@@ -85,11 +85,7 @@ export function MempoolFeePieChart({
   filteredTxTypeCounts,
   totalTxCount,
 }: {
-  filteredTxTypeCounts: {
-    token_transfer: number;
-    smart_contract: number;
-    contract_call: number;
-  };
+  filteredTxTypeCounts: Record<string, number>;
```

Comment:
Did we lose types here?

Reply:
addressed

Reply:
Yep, latest client has tx_type_counts defined as : { [p: string]: number }.